openbas-model/src/main/java/io/openbas/database/model/Executable.java

```diff
@@ -40,4 +40,12 @@ public Executable() {}
   public Executable(String id, String type, String name) {
     super(id, type, name);
   }
+
+  /*
+   * return the number of actions a given payload is expected to achieve
+   * by default this is 2 here, one file drop and one execution
+   */
+  public int getNumberOfActions() {
+    return 2;
```

Comment:
**thought:** this `2` can be considered a [magic number](https://stackoverflow.com/a/47902), a number that it is used directly in the code. 

to improve readability, one can introduce a constant, for example:
```
public static final int NUMBER_OF_ACTIONS_TO_ACHIEVE_BY_DEFAULT = 2;
```
or
```
public static final int DEFAULT_NUMBER_OF_ACTIONS_TO_ACHIEVE = 2;
```

```suggestion
  public int getNumberOfActions() {
    return DEFAULT_NUMBER_OF_ACTIONS_TO_ACHIEVE;
```

just to let you know that :)

Reply:
Thanks ! I fix it !